warehouse/query-core/src/test/java/datawave/query/testframework/QueryLogicTestHarness.java

```diff
@@ -86,9 +91,16 @@ public void assertLogicResults(BaseQueryLogic<Map.Entry<Key,Value>> logic, Colle
                         }
                     }
                     Assert.assertEquals(AttributeFactory.getKeepers(types), types);
+                } else {
+                    count++;
                 }
             }
             
+            // ignore empty documents (possible when only passing FinalDocument back)
+            if (count == 0) {
```

Comment:
I assume something from the updates in this pull request will return empty documents.